tests/unit_tests/scheduler/test_scheduler.py

```diff
@@ -270,3 +270,34 @@ async def init(iens, *args, **kwargs):
     assert sch.is_active()
     await execute_task
     assert not sch.is_active()
+
+
+@pytest.mark.timeout(6)
+async def test_that_long_running_jobs_were_stopped(storage, tmp_path, mock_driver):
+    killed_iens = []
+
+    async def kill(iens):
+        nonlocal killed_iens
+        killed_iens.append(iens)
+
+    async def wait(iens):
+        # all jobs with iens > 5 will sleep for 10 seconds and should be killed
+        if iens < 6:
+            await asyncio.sleep(0.5)
```

Comment:
will 0.1 also work and give faster test execution?

Reply:
Will try.